test/StringDiagram.lean

```diff
@@ -0,0 +1,56 @@
+import Mathlib.Tactic.Widget.StringDiagram
+
+/-! ## Example use of string diagram widgets -/
+
+section MonoidalCategory
+
+open ProofWidgets Mathlib.Tactic.Widget
+
+open CategoryTheory
+open scoped MonoidalCategory
+
+universe v u
+
+variable {C : Type u} [Category.{v} C] [MonoidalCategory C]
+
+lemma left_triangle {X Y : C} (η : 𝟙_ _ ⟶ X ⊗ Y) (ε : Y ⊗ X ⟶ 𝟙_ _) (w : False) :
+    η ▷ X ≫ (α_ _ _ _).hom ≫ X ◁ ε = (λ_ _).hom ≫ (ρ_ _).inv := by
+  /- Displays string diagrams for the both sides of the goal. -/
+  with_panel_widgets [StringDiagram]
+    /- Place the cursor here to see the string diagrams. -/
+    exact w.elim
```

Comment:
Can you leave a comment mentioning why the RHS has no string diagram

Reply:
This is because there are no non-structural morphisms in the RHS. I've tried to improve the message to make the situation clearer.

Probably a better way would be to display the trivial vertical strings instead of the text massage, but this would require some extra work.